Resources/Textures/Structures/Machines/computers.rsi/meta.json

```diff
@@ -2125,6 +2125,15 @@
           1
         ]
       ]
+    },
+    {
+      "name": "avionics-systems",
+      "directions": 1,
+      "delays": [
+        [
+          1
+        ]
+      ]
```

Comment:
I don't think direction needed as it's default, delays definitely not needed.

Reply:
Yeah, I was told to copy something, but I can see from looking at other examples of icon images that none of that was needed.